gossip/minimal/goshawk/main.go

```diff
@@ -71,6 +76,21 @@ func main() {
 			glog.Exitf("failed to create MySQL-based state manager: %v", err)
 		}
 	}
+	if len(*mySQLIncidentURI) > 0 {
+		glog.Infof("Incidents will be stored in %s", *mySQLIncidentURI)
+		db, err := sql.Open("mysql", *mySQLIncidentURI)
+		if err != nil {
+			glog.Exitf("failed to open MySQL incident database: %v", err)
```

Comment:
super-nit: Some of your logging statements start with a capital letter, others don't.